### DISCUSSION TITLE
Extracting translation manually is cumbersome, let's use automated tools for this

### DISCUSSION THREAD

**Discussion by lavocatt** [artemiscloud/activemq-artemis-self-provisioning-plugin#181] [Ideas]
There's tooling out there that could save us a ton of time and make us avoid mistakes: https://www.i18next.com/how-to/extracting-translations

WDYT?

**Comment by gaohoward** [comment]
+1

**Comment by lavocatt** [comment]
I created an issue for this https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/issues/206, closing the discussion.
